### PR TITLE
SDN-4405: 1.29_rebase:

### PR DESCRIPTION
**Pull request by jluhrsen**

### CODE REVIEW

File: cmd/cloud-network-config-controller/main.go [hunk 1]
```diff
@@ -268,7 +268,8 @@ func init() {
 	controllerNamespace = os.Getenv(controllerNamespaceEnvVar)
 	controllerName = os.Getenv(controllerNameEnvVar)
 	if controllerNamespace == "" || controllerName == "" {
-		klog.Exit("Controller ENV variables are empty: %q: %s, %q: %s, cannot initialize controller", controllerNamespaceEnvVar, controllerNamespace, controllerNameEnvVar, controllerName)
+		klog.Exit("Controller ENV variables are empty:", controllerNamespaceEnvVar, controllerNamespace, "%q:",
```
Comment (kyrtapz) [inline cmd/cloud-network-config-controller/main.go:271]: I think this is not correct, "%q:" will just get printed as is.
Can't we just do:
```
klog.Exitf("Controller ENV variables are empty: %q: %s, %q: %s, cannot initialize controller", controllerNamespaceEnvVar, controllerNamespace, controllerNameEnvVar, controllerName)
```
?

File: pkg/controller/node/node_controller.go [hunk 2]
```diff
@@ -108,8 +108,7 @@ func (n *NodeController) SyncHandler(key string) error {
 	// Skip synchronization if this node is still uninitialized by the Cloud Controller Manager,
 	// meaning that it still has taint cloudproviderapi.TaintExternalCloudProvider.
 	if taintKeyExists(node.Spec.Taints, cloudproviderapi.TaintExternalCloudProvider) {
-		klog.V(5).Info("Taint '%s' found on node, skipping until the node is ready",
-			cloudproviderapi.TaintExternalCloudProvider)
+		klog.V(5).Info("Taint", cloudproviderapi.TaintExternalCloudProvider, "found on node, skipping until the node is ready")
```
Comment (kyrtapz) [inline pkg/controller/node/node_controller.go:111]: NIT: Lets use Infof:
```
		klog.V(5).Infof("Taint '%s' found on node, skipping until the node is ready",
			cloudproviderapi.TaintExternalCloudProvider)
```

File: cmd/cloud-network-config-controller/main.go (hunk 1 above)
Comment (jluhrsen) [reply cmd/cloud-network-config-controller/main.go]: i over thought the errors. Exitf is all that was needed. updated.

File: pkg/controller/node/node_controller.go (hunk 2 above)
Comment (jluhrsen) [reply pkg/controller/node/node_controller.go]: i over thought the errors. Infof is all that was needed. updated.

**Pull request merged**